canonical-websites/design.ubuntu.com
# Remove web-style-guide from the site

## Done

Removed the old web style guide section and a reference to it from the /web page

## QA

- See that [/web-style-guide](http://localhost::8011/web-style-guide) no longer exists
- See that the section about the web style guide has been removed from  [/web](http://localhost::8011/web)

## Issue / Card

Fixes #66
